index.html

```diff
@@ -13,6 +13,9 @@
                 <ul>
               <li><a class="logolink" href="">My Logo</a></li>
               <li class="leftli"><a href=""><img class="imageone" src="hammenu.png" alt="ham"></a></li>
+              <li class="link-desktop"><a href="">Portfolio</a></li>
+              <li class="link-desktop"><a href=""> About</a></li>
+              <li class="link-desktop"><a href=""> Contact</a></li>
                 </ul>
             </nav>
         </header>
```

Comment:
- [ ] Kindly make sure the LinkedIn Icon is showing, I suggest checking the link. :recycle: 
![Screenshot from 2023-01-27 02-45-26](https://user-images.githubusercontent.com/97834160/214984124-836d99f7-2c1c-4caa-8eb3-38891a3d2a88.png)

Reply:
![image](https://user-images.githubusercontent.com/117091923/215057841-d760e868-951c-4ac8-9864-3ff263d25942.png)
![image](https://user-images.githubusercontent.com/117091923/215058406-4d291c7f-71fb-4122-8466-cac056264ef9.png)

Linked in images are working fine on my machine . Images also properly included in project files and its also added on Github repository and Pull request branch. if you can kindly please check again images are loading fine screen shot attached for your kind reference.